tests/testthat/test-infection_process.R

```diff
@@ -0,0 +1,139 @@
+test_that("infection_process does not schedule infections when no-one is infected", {
+skip("skip test")
```

Comment:
Should these be skipped? There's not a lot of point of merging in a file of non-working skipped tests. Was it intended to be introduced in a PR that adds a vignette?

Reply:
Sorry, I see now that this is waiting until #33 is resolved?